angularjs-portal-frame/src/main/webapp/portal/notifications/partials/notification-bell.html

```diff
@@ -0,0 +1,7 @@
+<div  ng-controller="NotificationController as notificationIconCtrl" ng-if="$storage.notificationsDemo">
+    <a title="click to view notifications" href="{{notificationIconCtrl.notificationUrl}}">
+        <div  class='notification-badge' ng-class='{ "notification-badge-empty" : notificationIconCtrl.isEmpty }'>
```

Comment:
Accessibility issue : empty / not-empty state is conveyed only visually (via `notification-badge-empty` class, which in turn drives a background color difference.  Should also convey this state in a way that will be read by screen readers.